### PR TITLE
Clicks will now save to the database, fixed authentication and database connection. Fixed cookies that where not working anymore

### PR DESCRIPTION
**Pull request by dhuber666**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -22,46 +22,54 @@ class App extends React.Component {
...
 	}
 
 	handlePanelClick() {
 		this.setState(prevState => ({ clickerCount: prevState.clickerCount + 1 }));
+		this.sendMessage();
 	}
 
 	render() {
 		const { clickerCount } = this.state;
+		const renderCount = clickerCount === null ? -1 : clickerCount;
```
[the remark this answers is not in the document]
Comment (charliearlie) [reply client/src/App.js]: To make this a little cleaner you can remove "=== null"

And have !clickerCount ? ...

Null is a false value so will equal false in this check.

**Comment by charliearlie** [comment]
Sorry, been really busy. Will fully review with the next hour

File: client/src/App.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhuber666) [reply client/src/App.js]: Good point!

**Comment by dhuber666** [comment]
No worries! Take your time 👍

File: client/.eslintrc [hunk 2]
```diff
@@ -9,7 +9,8 @@
 		"react/jsx-indent-props": [1, "tab"],
 		"jsx-a11y/anchor-is-valid": 0,
 		"react/no-did-mount-set-state": 0,
-		"linebreak-style": 0
+		"linebreak-style": 0,
+		"class-methods-use-this": 0
```
Comment (charliearlie) [inline client/.eslintrc]: Obviously, I must ask why this is necessary. I've noticed all your class methods in the front end use "this".

File: client/src/App.js [hunk 3]
```diff
@@ -22,46 +22,51 @@ class App extends React.Component {
...
 	}
 
 	handlePanelClick() {
 		this.setState(prevState => ({ clickerCount: prevState.clickerCount + 1 }));
+		this.sendMessage();
 	}
 
 	render() {
 		const { clickerCount } = this.state;
+		const renderCount = !clickerCount ? -1 : clickerCount;
```
Comment (charliearlie) [inline client/src/App.js]: I'm going to be so annoying with another upgrade on this. I'm reviewing on my phone so I missed the false option of the ternary.

I think `const renderCount = clickerCount || -1` ?

**Pull request merged**